docs/pages/versions/unversioned/sdk/video.mdx

```diff
@@ -18,6 +22,46 @@ import { SnackInline } from '~/ui/components/Snippet';
 
 <APIInstallSection />
 
+## Configuration in app.json/app.config.js
+
+You can configure `expo-video` using its built-in [config plugin](/config-plugins/introduction/). The plugin allows you to configure various properties that cannot be set at runtime and require building a new app binary to take effect.
+
+<ConfigPluginExample>
+
+  ```json app.json
+  {
+    "expo": {
+      "plugins": [
+        [
+          "expo-video",
+          {
+            "supportsBackgroundPlayback": true,
+            "supportsPictureInPicture": true
+          }
+        ]
+      ],
+    }
+  }
+  ```
+
+</ConfigPluginExample>
+
+<ConfigPluginProperties
+  properties={[
+    {
+      name: 'supportsBackgroundPlayback',
+      description: 'A boolean value to enable background playback on iOS. If `true` the `audio` key is added to the `UIBackgroundModes` array in the `Info.plist` file. When `false` the key is removed. When `undefined` the key is not modified.',
```

Comment:
```suggestion
      description: 'A boolean value to enable background playback on iOS. If `true`, the `audio` key is added to the `UIBackgroundModes` array in the &&Info.plist** file. If `false`, the key is removed. When `undefined`, the key is not modified.',
```

Let's use bold names for file names as per writing style guide. Using `,` above also improves readability by adding pauses in between based on boolean values.